setup.py

```diff
@@ -1,70 +1,66 @@
-# -*- coding: utf-8 -*-
 from setuptools import find_packages
 from setuptools import setup
 
 import sys
 
 
-version = '2.0.4.dev0'
+version = "2.0.4.dev0"
 
 additional_install_requires = []
 
-if sys.platform[:3].lower() == 'win':
-    additional_install_requires += ['nt_svcutils']
+if sys.platform[:3].lower() == "win":
+    additional_install_requires += ["nt_svcutils"]
```

Comment:
While you are at it, could you remove the `additional_install_requires` and put this directly in `install_requires` please?

```
nt_svcutils; platform_system == 'Windows',
```

Reply:
Will fix that too

Reply:
Is this dependency still valid? There is only one release from 2010 and the code in  e.g. in https://github.com/zopefoundation/nt_svcutils/blob/d8a19cf70d33d3a6d971f1a8f60b1e261c0caf8f/src/nt_svcutils/service.py#L245 is not Python 3 compatible, it es even a SyntaxError.

Reply:
As there was no interest in this package it got archived in February 2020.

Reply:
OK, then I recommend to drop the dependency here. But I do not have a Windows machine here to test the windows zeoctl services. We should setup a github action with this plattform for the tests.

Reply:
The nt_svcutils dependency was dropped.